Gemfile

```diff
@@ -1,5 +1,4 @@
 source 'https://rubygems.org'
-ruby '2.0.0'
```

Comment:
We already have it set on both files. Another fix would put it back in the Gemfile and test only against this same ruby version. The Catarse project (https://github.com/catarse/catarse) does this way. What do you think?